tests/unit/testsuite/Bolt/Boltpay/controllers/ShippingControllerTest.php

```diff
@@ -2,69 +2,318 @@
 
 require_once 'Bolt/Boltpay/controllers/ShippingController.php';
 
+/**
+ * Class Bolt_Boltpay_ShippingControllerTest
+ *
+ * Test the shipping controller, particularly with shipping and tax estimates and caching
+ */
 class Bolt_Boltpay_ShippingControllerTest extends PHPUnit_Framework_TestCase
```

Comment:
Great job adding tests 👍

Reply:
Thank you for the thorough review, as always!